pkg/email/templates/added_as_collaborator_body.template

```diff
@@ -0,0 +1,28 @@
+<!DOCTYPE  html PUBLIC "-//W3C//DTD XHTML 1.0 Transitional//EN" "http://www.w3.org/TR/xhtml1/DTD/xhtml1-transitional.dtd">
+<html xmlns="http://www.w3.org/1999/xhtml" xml:lang="en" lang="en">
+   <head>
+      <meta http-equiv="Content-Type" content="text/html; charset=utf-8"/>
+      <title>Content</title>
+      <style type="text/css"> * {margin:0; padding:0; text-indent:0; }
+         h1 { color: #1A1A1A; font-family:Arial, sans-serif; font-style: normal; font-weight: bold; text-decoration: none; font-size: 39pt; }
+         .s1 { color: #707579; font-family:Arial, sans-serif; font-style: normal; font-weight: normal; text-decoration: none; font-size: 15.5pt; }
+         p { color: #1A1A1A; font-family:Arial, sans-serif; font-style: normal; font-weight: normal; text-decoration: none; font-size: 15.5pt; margin:0pt; }
+         h2 { color: #005BA0; font-family:Arial, sans-serif; font-style: normal; font-weight: bold; text-decoration: none; font-size: 15pt; }
+         .a { color: #005BA0; font-family:Arial, sans-serif; font-style: normal; font-weight: normal; text-decoration: none; font-size: 15.5pt; }
+         .s2 { color: #1A1A1A; font-family:Arial, sans-serif; font-style: normal; font-weight: normal; text-decoration: none; font-size: 15.5pt; }
+      </style>
+   </head>
+   <body>
+      <h1 style="padding-top: 3pt;padding-left: 5pt;text-indent: 0pt;text-align: left;">MINT</h1>

```

Comment:
`Non-Blocking` Thanks for working on the formatting changes for the email @OddTomBrooks ! I think it looks a lot closer. It's not quite matching [FIGMA](https://www.figma.com/file/6tkIhfNfcsKEAkqXo1qvXH/MINT---Read-Only-View?node-id=902%3A73536), but I think that we can say it's close enough for now. We might consider requesting some front end assistance dialing in the final formatting to make sure the font, and font size matches and that we are following USWDS. There are a few other things I noticed, like figma has both words in `unfollow it` as the url. I think we can consider that part of a final pass ticket in the future.